engines/dfc_provider/app/serializers/dfc_provider/enterprise_serializer.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Serializer used to render a DFC Enterprise from an OFN Enterprise
+# into JSON-LD format based on DFC ontology
+module DfcProvider
+  class EnterpriseSerializer < ActiveModel::Serializer
+    attribute :id, key: '@id'
+    attribute :type, key: '@type'
+    attribute :vat_number, key: 'dfc:VATnumber'
+    has_many :defines, key: 'dfc:defines'
+    has_many :supplies,
+             key: 'dfc:supplies',
+             serializer: DfcProvider::SuppliedProductSerializer
+    has_many :manages,
+             key: 'dfc:manages',
+             serializer: DfcProvider::CatalogItemSerializer
+
+    def id
+      dfc_provider_routes.api_dfc_provider_enterprise_url(
+        id: object.id,
+        host: root_url
+      )
+    end
+
+    def type
+      'dfc:Entreprise'
+    end
+
+    def vat_number; end
+
+    def defines
+      []
+    end
+
+    def supplies
+      Spree::Variant.
+        joins(product: :supplier).
+        where('enterprises.id' => object.id)
+    end
+
+    def manages
+      Spree::Variant.
+        joins(product: :supplier).
+        where('enterprises.id' => object.id)
```

Comment:
I see `#supplies` and `#manages` are identical here. Is that right?

Reply:
`#supplies` is using the `SuppliedProductSerializer` and `#manages` is using `CatalogItemSerializer`. The catalog items are what's being sold (distributed), right?

Reply:
Yes exactly, SuppliedProduct is what you have in your hand, CatalogItem is what you will actually sell (and so linked to offers).